UnityProjects/MRTKDevTemplate/Packages/packages-lock.json

```diff
@@ -179,7 +187,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.collab-proxy": {
-      "version": "1.15.18",
+      "version": "1.17.0",
```

Comment:
Did you mean to check these in? This usually happens when you're opening the project in a newer version of Unity

Reply:
i think these happened when i used the latest 2020 lts

Reply:
i can revert